datalab/datalab.sh

```diff
@@ -38,6 +38,9 @@ if [[ "${ROLE}" == 'Master' ]]; then
     echo 'spark.sql.warehouse.dir=/root/spark-warehouse' >> "${SPARK_CONF}"
   fi
 
+  DATALAB_DIR="${HOME}/datalab"
```

Comment:
> Should there be a metadata key of a GCS directory to copy here?

I don't know of any reason to add such a thing, but maybe I'm overlooking something.

What makes you ask?